examples/sprite_camera_follow/main.rs

```diff
@@ -0,0 +1,182 @@
+        let x_move = input.axis_value("entity_x").unwrap();
+        let y_move = input.axis_value("entity_y").unwrap();
+
+        for (_, player) in (&_asdf, &mut transforms).join() {
+            player.translation.x += x_move as f32 * 5.0;
+            player.translation.y += y_move as f32 * 5.0;
+        }
+    }
+}
+
+fn load_sprite_sheet(world: &mut World) -> SpriteSheetHandle {
+    let res_path = format!(
+        "{}/examples/sprite_camera_follow/resources",
+        application_root_dir()
+    );
+    let texture_handle = {
+        let loader = world.read_resource::<Loader>();
+        let texture_storage = world.read_resource::<AssetStorage<Texture>>();
+        loader.load(
+            format!("{}/Circle.png", res_path),
+            PngFormat,
+            TextureMetadata::srgb_scale(),
+            (),
+            &texture_storage,
+        )
+    };
+    let texture_id = 0;
+    let mut material_texture_set = world.write_resource::<MaterialTextureSet>();
+    material_texture_set.insert(texture_id, texture_handle);
+
+    let loader = world.read_resource::<Loader>();
+    let sprite_sheet_store = world.read_resource::<AssetStorage<SpriteSheet>>();
+    loader.load(
+        format!("{}/Circle.ron", res_path),
+        SpriteSheetFormat,
+        texture_id,
+        (),
+        &sprite_sheet_store,
+    )
+}
+
+fn init_sprite(world: &mut World, sprite_sheet: &SpriteSheetHandle) -> Entity {
+    let mut transform = Transform::default();
+    transform.translation.x = 100.0;
+    transform.translation.y = 0.0;
+    let sprite = SpriteRender {
+        sprite_sheet: sprite_sheet.clone(),
+        sprite_number: 1,
+        flip_horizontal: false,
+        flip_vertical: false,
+    };
+    world.create_entity().with(transform).with(sprite).build()
+}
+
+fn init_player(world: &mut World, sprite_sheet: &SpriteSheetHandle) -> Entity {
+    let mut transform = Transform::default();
+    transform.translation.x = 0.0;
+    transform.translation.y = 0.0;
+    let sprite = SpriteRender {
+        sprite_sheet: sprite_sheet.clone(),
+        sprite_number: 0,
+        flip_horizontal: false,
+        flip_vertical: false,
+    };
+    world
+        .create_entity()
+        .with(transform)
+        .with(Player)
+        .with(sprite)
+        .build()
+}
+
+fn init_camera(world: &mut World, parent: Entity) {
+    let mut transform = Transform::default();
+    transform.translation.z = 1.0;
+    world
+        .create_entity()
+        .with(Camera::from(Projection::orthographic(
+            -250.0, 250.0, 250.0, -250.0,
+        ))).with(Parent { entity: parent })
+        .with(transform)
+        .build();
+}
+
+struct Example;
+
+impl<'a, 'b> SimpleState<'a, 'b> for Example {
+    fn on_start(&mut self, data: StateData<GameData>) {
+        let world = data.world;
+        let sprite_sheet_handle = load_sprite_sheet(world);
+
+        let _anchor = init_sprite(world, &sprite_sheet_handle);
```

Comment:
I assume this sprite is meant to allow seeing the player's movement?
Maybe add a little comment here:
`// A sprite with fixed position, to see how the player moves`

Reply:
Yes it is, I'll add a comment